src/components/PageButton.vue

```diff
@@ -21,13 +19,14 @@ const props = defineProps({ button: { type: Object, required: true } })
 <style lang="scss" scoped>
 button {
   background-color: transparent;
-  color: #ffff;
-  opacity: 60%;    
+  color: rgb(160, 160, 160);
+  fill:rgb(160, 160, 160);
   cursor: pointer;
```

Comment:
If you aren't going to use an opacity, then use a hex. If you're going to use opacity then use `rgba`